packages/fast-components-styles-msft/src/number-field/index.ts

```diff
@@ -13,6 +13,11 @@ const styles: ComponentStyles<NumberFieldClassNameContract, DesignSystem> = {
             format("0 6px 0 {0}", horizontalSpacing()),
             format("0 {0} 0 6px", horizontalSpacing())
         ),
+        "&::-webkit-inner-spin-button": {
+            appearance: "none",
+            "-webkit-appearance": "none",
+            "-moz-appearance": "none",
```

Comment:
Firefox does not support `-webkit-inner-spin-button`.
Theoretically assigning it an appearance within this selector will not affect how the control looks on firefox.

Reply:
Yeah, I'll get this updated. I saw that it was changing the appearance in FF. Still need to land this with others.